idd/Energy+.idd.in

```diff
@@ -18632,6 +18632,28 @@ SurfaceProperty:SolarIncidentInside,
        \type object-list
        \object-list ScheduleNames
 
+SurfaceProperty:IncidentSolarMultiplier,
+      \min-fields 3
+  A1, \field Surface Name
+      \required-field
+      \type object-list
+      \object-list SurfaceNames
+      \note Enter the name of an exterior window outside surface object
+  N1, \field Incident Solar Multiplier
+      \note a constant multiplier for window solar transmittance
+      \note and visible transmittance. If the Shading Multiplier Schedule Name is
+      \note defined, the product of these two will be the final shading multiplier.
+      \type real
+      \units dimensionless
+      \minimum 0
+      \maximum 1
```

Comment:
I know this project started out as providing a simplified shading solution.  But the final approach is fairly general and it could be used to amplify incident solar as well as reduce it.  It seems to me that the maximum limit of 1.0 is a bit arbitrary.  If you want to allow magical reduction of solar, why not allow magical increases as well? E.g. users could quickly see what would happen if the windows transmitted 20% more solar.  Or maybe they know a neighboring building is reflecting a ton of solar during certain times.